scripts/generate-import.sh

```diff
@@ -1,16 +1,29 @@
-#!/usr/bin/env bash
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p yq jq curl
```

Comment:
hub?

Reply:
Wouldn't help, since it doesn't use it here. There's no `hub login`, sadly.